testsuite/oauth/authorize.py

```diff
@@ -156,7 +158,12 @@ def _execute_step(self, step):
         if not elem.is_displayed() and step.get('optional'):
             return
         elif not elem.is_displayed():
-            raise ElementNotFoundException('Element is hidden', self.browser)
+            try:
+                wait = WebDriverWait(self.browser, VISIBILITY_TIMEOUT)
```

Comment:
Did something change in one of the existing portals that made the previous approach break?